rc/tools/spell.kak

```diff
@@ -37,7 +39,7 @@ define-command -params ..1 -docstring %{
         fi
 
         {
-            sed 's/^/^/' "$kak_opt_spell_tmp_file" | eval "aspell --byte-offsets -a $options" 2>&1 | awk '
+            sed 's/^/^/' "$kak_opt_spell_tmp_file" | eval "$kak_opt_spell_cmd $options" 2> /dev/null | awk "$kak_opt_spell_awk_options" '
```

Comment:
We want errors to be reported, not ignored, plus that change is made in an unrelated hunk.

Reply:
That was accidental, I do not want that either.